static/global.css

```diff
@@ -1,5 +1,9 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700");
 
+html, body {
+  overflow-x: hidden;
```

Comment:
This isn't important, but I think that if we need this, then we have other components to fix instead of adding `overflow: hidden`.

Reply:
I agree that we have other components to fix on the index page. As of right now, the layout is pretty solid for once using the `overflow-x: hidden;` style which is why I left it here temporarily. I have created issue #63 to look into what is causing the horizontal scroll on smaller screen sizes.